<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <widget class="QRadioButton" name="radioButton_NIWRITE">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(117, 142, 79);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>NIWrite</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="radioButton_WRITE">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(81, 152, 114);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Write</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="radioButton_NIREAD">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(153, 178, 221);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>NIRead</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QPushButton" name="pushButton_ADD">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="packetViewer" name="treeWidget_RESPONSE">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column>
       <property name="text">
        <string>Packet content</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>#t</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>#w</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QRadioButton" name="radioButton_RMWSUM">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 177, 64);</string>
      </property>
      <property name="text">
       <string>RMWsum</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QProgressBar" name="progressBar_WORDS">
      <property name="maximum">
       <number>368</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string notr="true">%v</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_ANDTERM">
      <property name="text">
       <string>AND term:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEdit_ANDTERM">
      <property name="inputMask">
       <string>&gt;HHHHHHHH</string>
      </property>
      <property name="text">
       <string>00000000</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="packetViewer" name="treeWidget_PACKET_WIEVER">
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>26</number>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>100</number>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">Packet content</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">#t</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">#w</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QRadioButton" name="radioButton_RMWBITS">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(247, 240, 109);

color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>RMWbits</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>OR term:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>nWords:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEdit_ORTERM">
      <property name="inputMask">
       <string>&gt;HHHHHHHH</string>
      </property>
      <property name="text">
       <string>00000000</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEdit_NWORDS">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioButton_READ">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(141, 128, 173);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEdit_ADDRESS">
      <property name="inputMask">
       <string>&gt;HHHHHHHH</string>
      </property>
      <property name="text">
       <string>00000000</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QProgressBar" name="progressBar_WORDS_EXPECTED">
      <property name="maximum">
       <number>368</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="format">
       <string notr="true">%v</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QPushButton" name="pushButton_SEND">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>packetViewer</class>
   <extends>QTreeWidget</extends>
   <header location="global">packetviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
